translations/locales/fr-CA/translations.json

```diff
@@ -0,0 +1,591 @@
+{
+    "Nav": {
+      "File": {
+        "Title": "Fichier",
+        "New": "Nouveau",
+        "Share": "Partager",
+        "Duplicate": "Dupliquer",
+        "Open": "Ouvrir",
+        "Download": "Télécharger",
+        "AddToCollection": "Ajouter à la collection",
+        "Examples": "Exemples"
+      },
+      "Edit": {
+        "Title": "Edition",
+        "TidyCode": "Nettoyer le code",
+        "Find": "Rechercher",
+        "FindNext": "Prochaine correspondance",
+        "FindPrevious": "Correspondance précédente"
```

Comment:
I think since you worked on this, "Replace" got added as an option, and Find Next/Find Previous got removed (see English for reference: https://github.com/processing/p5.js-web-editor/blob/develop/translations/locales/en-US/translations.json#L17). Could you add this translation?